packages/ui/src/components/TextLink/index.tsx

```diff
@@ -28,7 +59,11 @@ const TextLink = ({ className, children, text, icon, type = 'primary', to, ...re
   }
 
   return (
-    <a className={classNames(styles.link, styles[type], className)} {...rest} rel="noreferrer">
+    <a
+      className={classNames(styles.link, styles[type], className)}
+      {...hyperLinkProps}
+      rel="noreferrer"
```

Comment:
i think `noopener` is good? because we do want to keep referrer sometimes

Reply:
updated